zeplin/Sources/Screen/ScreenViewController.swift

```diff
@@ -20,6 +20,16 @@ final class ScreenViewController: UIViewController, ios_toolkit.View, ErrorDispl
     private(set) var completionObservable = PublishSubject<Void>()
     private(set) var loadingView: LoadingView
     
+    var statusBarHidden = true {
+      didSet(newValue) {
```

Comment:
Can you use RxSwift instead of overriding `didSet`? Example:

```swift
let statusBarStyleRelay = BehaviorRelay<UIStatusBarStyle>(value: .default)

override var preferredStatusBarStyle: UIStatusBarStyle {
    return statusBarStyleRelay.value
}

func setupBindings(for statusBarObservable: Observable<UIStatusBarStyle>) {
    statusBarObservable
        .bind(to: statusBarStyleRelay)
        .disposed(by: disposeBag)

    statusBarStyleRelay
        .distinctUntilChanged()
        .do(onNext: { [weak self] _ in
            self?.setNeedsStatusBarAppearanceUpdate()
        })
        .subscribe()
        .disposed(by: bag)
}
```